### PR TITLE
add ReconnectAction and add it in status bar popup

### LINKED ISSUE #398: CLion plugin: add `Reconnect` button/action
**Description**
When plugin is up, and server was restarted it may take a long time (3-5 min) to connect again to server. It could be solved by changing connection settings to different ones and then back again. It would be much better to have some button/action that does 
it for user

**Expected behavior**
When server is restarted or launched, user can click/invoke `Reconnect` button/action to refresh plugin for quick reconnection

### PR DESCRIPTION
**Pull request by vol0n**
closes #398

### CODE REVIEW

File: clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/client/Client.kt [hunk 1]
```diff
@@ -72,6 +73,7 @@ class Client(
         if (isDisposed) {
             // if client is disposed, then connection settings were changed, and requests issued to this client
             // are no longer relevant, so we don't execute them
+            notifyWarning("Reconnecting to server! Request won't be executed! Please try again, later.")
```
Comment (alexey-utkin) [inline clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/client/Client.kt:76]: Need localization in UTBot.properties?

File: clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/actions/ReconnectAction.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import org.utbot.cpp.clion.plugin.UTBot
+import org.utbot.cpp.clion.plugin.client.ClientManager
+
+class ReconnectAction: AnAction(UTBot.message("actions.reconnect")) {
+    override fun actionPerformed(e: AnActionEvent) {
+        e.project!!.service<ClientManager>().restartClient()
+    }
+
+    override fun update(e: AnActionEvent) {
+        e.presentation.isEnabledAndVisible = e.project != null
```
Comment (alexey-utkin) [inline clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/actions/ReconnectAction.kt:15]: Do we need to extend this availability by ON/OFF boolean from per-project UTBotCpp plugin option?

Comment (vol0n) [reply clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/actions/ReconnectAction.kt:15]: I didn't add ON/OFF functionality for plugin yet. So no

File: clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/client/Client.kt (hunk 1 above)
Comment (vol0n) [reply clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/client/Client.kt]: fixed

**Review by alexey-utkin — APPROVED** [review]

**Pull request merged**